falcon_mutualauth/mutualauth.py

```diff
@@ -41,9 +41,11 @@ class MutualAuthRequest(Request):
 
     def _inject_headers(self):
         peer_cert = self.channel.transport.getPeerCertificate()
-        common_name = peer_cert.get_subject().CN
+        user = peer_cert.get_subject().CN
 
-        roles = self.roles_map.get(common_name, [])
+        roles = self.roles_map.get(user, [])
```

Comment:
Doh! Should have used the Account-Service example! https://github.com/RackSec/account-service/blob/master/src/account_service/config.py#L71-L80